Resources/Prototypes/weather.yml

```diff
@@ -0,0 +1,246 @@
+- type: weather
+  id: Ashfall
+  sprite:
+    sprite: /Textures/Effects/weather.rsi
+    state: ashfall
+  tiles:
+    - FloorCave
+    - FloorCaveDrought
+    - FloorGrassLight
+    - FloorGrassDark
+    - FloorGrassJungle
+    - FloorDirt
+    - FloorGrass
+    - Lattice
```

Comment:
Some weather (e.g. radstorm) may want to go over specific tiles so wasn't sure how to handle that given the weather side is driving it rather than the tile side.